### PR TITLE
fix: Trailing slash removal

### PR DESCRIPTION
**Pull request by dj-stormtrooper**

### CODE REVIEW

**Comment by yevgenypats** [comment]
I can't recall now why we changed it to `false` can we find the commit history because I think before that we had a different bug and this why I created this config.

**Comment by dj-stormtrooper** [comment]
> I can't recall now why we changed it to `false` can we find the commit history because I think before that we had a different bug and this why I created this config.

Ok, I'll check!

BTW `trailingSlashes: true` is default behaviour

File: docs/developers/sdk/table/fetch-resolvers.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The `TableResolver` is flexible allowing you to define your own pagination logic
 The collector in the SDK is slice-aware, so if you have a slice of resources, you can just push the slice as a whole, without iterating.
 :::
 
-Here's an example from the [provider template](https://github.com/cloudquery/cq-provider-template/blob/main/resources/demo_resource.go):
+Here's an example from the [provider template](https://github.com/cloudquery/cq-provider-template/blob/main/resources/services/demo/resource.go):
```
Comment (dj-stormtrooper) [inline docs/developers/sdk/table/fetch-resolvers.md:20]: Need an expert from OSS team to check if it's a correct link)

Also should we rewrite this code-snippet below?

**Comment by dj-stormtrooper** [comment]
@bbernays Can you check newest changes?

@yevgenypats I checked the history of `docusaurus.config.js` and also tried to find something in repo history - nothing found

File: docs/developers/sdk/table/fetch-resolvers.md (hunk 1 above)
Comment (yevgenypats) [reply docs/developers/sdk/table/fetch-resolvers.md:20]: seems correct what you suggested

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**